extensions/amp-story/1.0/page-advancement.js

```diff
@@ -546,16 +546,16 @@ export class ManualAdvancement extends AdvancementConfig {
   }
 
   /**
-   * Checks if click was inside of one of the side edges of the screen.
+   * Checks if click was inside of one of the side edges of the story page.
    * @param {!Event} event
-   * @param {!ClientRect} pageRect
    * @return {boolean}
    * @private
    */
-  isInScreenSideEdge_(event, pageRect) {
+  isInScreenSideEdge_(event) {
```

Comment:
Could we change the name of this method to `isInStoryPageSideEdge_()` or something similar?

Reply:
Good point, fixed!